File: SPUIKitCategory/UIWindow/UIWindow+KeyWindow.m
//
//  UIWindow+KeyWindow.m
//  HeartHeart
//
//  Created by LSP on 15/7/3.
//  Copyright (c) 2015年 LSP. All rights reserved.
//

#import "UIWindow+KeyWindow.h"

static UITableView *reportTableView;
__weak UIView *reportView;
@implementation UIWindow (KeyWindow)
+ (instancetype)window{
    return [UIApplication sharedApplication].keyWindow;
}

@end

File: Sources/UIWindow/UIWindow+KeyWindow.swift
//
//  UIWindow+KeyWindow.swift
//  SPUIKitCategory module
//
//  Created by LSP on 2020/12/22.
//  Copyright © 2020 LSP. All rights reserved.
//

#if !os(macOS)
import UIKit

extension UIWindow {
    public
    class var window: UIWindow {
        get {
            if let v = UIApplication.shared.keyWindow {
                return v
            }
            if let v = UIApplication.shared.windows.first {
                return v
            }
            return UIWindow()
        }
    }
}
#endif
